README.md

```diff
@@ -24,4 +24,7 @@ To create all necessary tables in database, run:
 ## Code formatting
 Before committing, format code using `black` formater:
 
-```black .```
\ No newline at end of file
+```black .```
+
+## Text extraction
+To run textract package on Windows you need to add `c:\Program Files\Git\mingw64\bin\` to your PATH variables.
```

Comment:
How to install on Linux/Mac?  Check similar request here:
![image](https://user-images.githubusercontent.com/32220997/101170479-cf605000-363e-11eb-98e4-ee3437d54232.png)